sw/device/silicon_creator/lib/cert/dice.c

```diff
@@ -86,9 +111,33 @@ static void curr_tbs_signature_le_to_be_convert(void) {
                    kAttestationSignatureBytes / 2);
 }
 
+/**
+ * Helper function to compute measurements of various OTP paritions that are to
+ * be included in attestation certificates.
+ */
+static status_t measure_otp_partition(otp_partition_t partition,
+                                      hmac_digest_t *measurement) {
+  uint32_t msg[kDiceMeasuredOtpPartitionMaxSizeIn32bitWords] = {0};
```

Comment:
How large can this be? If this value gets large, this could take a lot of space on the stack. Maybe reading and hashing in several goes using a small buffer would be safer?

Reply:
fixed.